internal/npm_install/npm_install.bzl

```diff
@@ -130,6 +139,14 @@ filegroup(
           repository_ctx.attr.yarn_lock,
           repository_ctx.path("yarn.lock"))
 
+  # Add data dependencies to the repository
```

Comment:
nit: seems like the code duplication between npm and yarn is getting to the point where it's worth sharing

Reply:
good call. factored out the common code.